src/SpectatorView.Unity/Assets/SpectatorView.Editor/Shaders/Blur.shader

```diff
@@ -0,0 +1,83 @@
+﻿Shader "SV/Blur"
+{
+    Properties
+    {
+        _MaskTexture("MaskTexture", 2D) = "white" {}
+        _BlurIntensity("BlurIntensity", float) = 0.00375
+    }
+    SubShader
+    {
+        // No culling or depth
+        Cull Off ZWrite Off ZTest Always
+
+        Pass
+        {
+            CGPROGRAM
+            #pragma vertex vert
+            #pragma fragment frag
+
+            #include "UnityCG.cginc"
+
+            struct appdata
+            {
+                float4 vertex : POSITION;
+                float2 uv : TEXCOORD0;
+            };
+
+            struct v2f
+            {
+                float2 uv : TEXCOORD0;
+                float4 vertex : SV_POSITION;
+            };
+
+            v2f vert (appdata v)
+            {
+                v2f o;
+                o.vertex = UnityObjectToClipPos(v.vertex);
+                o.uv = v.uv;
+                return o;
+            }
+
+            Texture2D _MaskTexture;
+            float _BlurIntensity;
+            SamplerState sampler_point_clamp;
+
+            fixed4 frag(v2f i) : SV_Target
+            {
+                half4 blurVal = fixed4(0,0,0,0);
+                float sumX = 0.0;
+
+                sumX += _MaskTexture.Sample(sampler_point_clamp, float2(i.uv[0] - 5.0 * _BlurIntensity, i.uv[1])).r * 0.025;
```

Comment:
>0.025 [](start = 118, length = 5)

If these magic numbers mean anything or came from somewhere, can we reference that in a comment?

Reply:
Added comment